test/format/format.sh

```diff
@@ -591,6 +591,8 @@ resolve()
 
 # Start a single job.
 count_jobs=0
+test_memory_max=$(free -m | grep -oP '\d+' | head -n 1)
```

Comment:
Is it better to use the available memory or the total memory? I think the command here uses the total memory.

Reply:
I would be better to use total memory in this case. The problem with using available memory is that it would hold inconsistent results when the format script is already running and we are trying to spin up a new test/format run.  The format script doesn't run in batches but instead runs a test/format whenever it gets the chance too. This means that a concurrent test/format could be already taking up 25% of the memory.